### ISSUE TITLE
Lack of fixture functionality

### ISSUE THREAD

**Issue by dlowe-net** [joaotavora/fiasco#34]
In the original stefil, suites also did double duty as fixtures, routines to set up and tear down the global environment. This was done by defining the suite with a body in which `run-child-tests` was called. The body argument was omitted on the implementation rewrite, but with no replacement functionality.

There's a number of options to support setup and teardown for tests, and I don't know which one is right for fiasco. It's tough having none of them, though. I'd rather not wrap `deftest` in my own macro.

**Comment by joaotavora** [comment]
yes, indeed. I took that out because I found it overlay complicated.

In the meantime, some time ago I re-implemented a fiasco clone from scratch in a single file, that I just commited.  `mfiasco.lisp`.  Simpler to understand, has suites, and takes away all the cruft that I personally didn't like.  It some fixture support.  Probably missing docs, but I tried, more or less.  Best I can do, for now.

But it does exemplify how I would like fixture support to come back to Fiasco

**Comment by joaotavora** [comment]
```
(defmacro defsetup (package &body body)
  "Define tests in PACKAGE to need the setup of BODY."
 ...)

(defmacro defteardown (package &body body)
  "Define tests in PACKAGE to need the teardown of BODY."
...)
```

This is how it works in mfiasco.  I suppose something like this could be added to fiasco, too.
